index.html

```diff
@@ -53,95 +56,108 @@ <h1 class="first-introduction">I'm Raphael.<br /> I am so glad to see you!</h1>
-                    <li><label class="tool-press">HTML</label></li>
-                    <li><label class="tool-press">CSS</label></li>
-                    <li><label class="tool-press">JAVASCRIPT</label></li>
-                </ul>
-                <button class="see-project">See Project</button>
             </div>
         </div>
         <!-- End of Project 1 -->
 
         <!-- Start of Project 2 -->
         <div id="tonic-project">
             <div class="data">
-                <img src="./images/snapshots/Snapshoot Portfolio 2.png" alt="Tonic project snapshot">
-                <h2 class="project-title">Multi-Post <br />Stories</h2>
-                <div class="work-nature">
-                    <label class="canopy">CANOPY</label>
-                    <div class="bullet-point"><img src="./images/icons/Counter.png" alt=""></div>
-                    <label class="backend"> Back End Dev</label>
-                    <div class="bullet-point"><img src="./images/icons/Counter.png" alt=""></div>
-                    <label class="year">2015</label>
+                <div class="image-div"><img src="./images/snapshots/Snapshoot Portfolio 2.png" alt="Tonic project snapshot"></div>
+                <div class="desktop-image-div-2"><img class="desktop-snapshop" src="./images/desktop/Snapshoot-Desktop2.png" alt="Tonic project snapshot"></div>
+                <div class="project-content-2">
+                    <h2 class="project-title">Multi-Post <br />Stories</h2>
+                    <div class="work-nature">
+                        <label class="canopy">CANOPY</label>
+                        <div class="bullet-point"><img src="./images/icons/Counter.png" alt=""></div>
+                        <label class="backend"> Back End Dev</label>
+                        <div class="bullet-point"><img src="./images/icons/Counter.png" alt=""></div>
+                        <label class="year">2015</label>
+                    </div>
+                    <p class="work-description">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    <ul class="tools">
+                        <li><label class="tool-press">HTML</label></li>
+                        <li><label class="tool-press">CSS</label></li>
+                        <li><label class="tool-press">JAVASCRIPT</label></li>
+                    </ul>
+                    <button class="see-project">See Project</button>
                 </div>
-                <p class="work-description">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
-                <ul class="tools">
-                    <li><label class="tool-press">HTML</label></li>
-                    <li><label class="tool-press">CSS</label></li>
-                    <li><label class="tool-press">JAVASCRIPT</label></li>
-                </ul>
-                <button class="see-project">See Project</button>
             </div>
         </div>
```

Comment:
- [ ] Kindly note that for every second project, the work image and project description are flipped. You can use `flex-direction: row-reverse;` or any other method you see fit. Kindly flip them to match the template.
![image](https://user-images.githubusercontent.com/94551484/229280681-43ca6db2-4d0b-423f-a509-2049b5e626db.png)